code/online_challenges/src/codechef/CASH/hard_cash.java

```diff
@@ -0,0 +1,50 @@
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+class Codechef
```

Comment:
Code looks fine, but can you just clean up the formatting a bit? Use an online Java formatter.

Reply:
I cleaned up the formatting. plz review @arnavb